README.md

```diff
@@ -24,7 +24,8 @@ All of the arguments to a function must be annotated with one of the supported p
 - Textarea
 - Integer
 - Number
-- Select
+- Autocomplete
```

Comment:
Nit: You have this statement on the top

All of the arguments to a function must be annotated with one of the supported parameter types. Currently there are 5.

Maybe reword `Currently there are 5` to `The current supported ones are` so that people don't need to change how many there are when new ones are added.

Reply:
good point, thanks